modules/cats/shared/src/test/scala/eu/timepit/refined/cats/CatsSpec.scala

```diff
@@ -11,16 +12,26 @@ class CatsSpec extends Properties("cats") {
     val refTypeOrder: Unit = () // shadow the `Order` instance so the `Eq` instance is tested
     locally(refTypeOrder) // prevent a "unused" warning
 
-    PosInt.unsafeFrom(5) === PosInt.unsafeFrom(5)
```

Comment:
I'm using `unsafeFrom` in the tests to avoid the overhead of the `RefineMacro` so that test code compiles faster.

Reply:
Ah, ok, I'll revert